bookclub-frontend/src/services/bookclubServices.js

```diff
@@ -22,8 +22,10 @@ const createToken = async () => {
 
 export const searchByCategory = async (searchInput) => {
   try {
-    const res = await axios.get(`${url}/books/search/${searchInput}`);
-    return res.data;
+    await axios.get(`${url}/books/search/${searchInput}`)
```

Comment:
keep in mind for later: if we still come across errors with these requests, we can start by removing `try` and just making axios requests the classic way. Another route is possibly not needing the `async/await`, but I can't see how they'd do any harm for being there, it's just a thought for if we run into issues later